src/main/java/me/leoko/advancedban/manager/CommandManager.java

```diff
@@ -117,19 +120,19 @@ public void onCommand(final Object sender, final String cmd, final String[] args
                                 }
                             }
 
-                            if ((mi.isOnline(args[0]) && mi.hasPerms(mi.getPlayer(args[0]), "ab." + pt.getName() + ".exempt")) ||
-                                    Universal.get().isExemptPlayer(args[0])) {
+                            if ((mi.isOnline(args[0]) && mi.hasPerms(mi.getPlayer(args[0]), "ab." + pt.getName() + ".exempt"))
+                                    || Universal.get().isExemptPlayer(args[0])) {
                                 MessageManager.sendMessage(sender, pt.getBasic().getConfSection() + ".Exempt", true, "NAME", args[0]);
                                 return;
                             }
 
-                            if ((pt.getBasic() == PunishmentType.MUTE && PunishmentManager.get().isMuted(uuid)) ||
-                                    (pt.getBasic() == PunishmentType.BAN && PunishmentManager.get().isBanned(uuid))) {
+                            if ((pt.getBasic() == PunishmentType.MUTE && PunishmentManager.get().isMuted(uuid))
+                                    || (pt.getBasic() == PunishmentType.BAN && PunishmentManager.get().isBanned(uuid))) {
                                 MessageManager.sendMessage(sender, pt.getBasic().getConfSection() + ".AlreadyDone", true, "NAME", args[0]);
                                 return;
                             }
 
-                            new Punishment(name, uuid, reason != null ? reason.toString() : null, mi.getName(sender), isTemp && end == -1 ? PunishmentType.BAN : pt, TimeManager.getTime(), end, isTemp && args[1].matches("#.+") ? args[1].substring(1) : null, -1).create();
+                            new Punishment(name, uuid, reason != null ? reason.toString() : "none", mi.getName(sender), isTemp && end == -1 ? PunishmentType.BAN : pt, TimeManager.getTime(), end, isTemp && args[1].matches("#.+") ? args[1].substring(1) : null, -1).create();
```

Comment:
The reason has to be null as the getReason() method checks if it's null and if so returns the default reason:
https://github.com/Beelzebu/AdvancedBan/blob/1f18591de0d3617fa63d596d7a6330c73d3c2981/src/main/java/me/leoko/advancedban/utils/Punishment.java#L50